src/ConfigWidgets/SymbolLocationsDialog.cpp

```diff
@@ -189,7 +202,10 @@ void SymbolLocationsDialog::OnAddFolderButtonClicked() {
   if (directory.isEmpty()) return;
 
   settings.setValue(kFileDialogSavedDirectoryKey, directory);
-  ErrorMessageOr<void> result = TryAddSymbolPath(std::filesystem::path{directory.toStdString()});
+
+  ScopedMetric metric{metrics_uploader_, OrbitLogEvent::ORBIT_SYMBOL_LOCATIONS_ADD_FOLDER};
+  ErrorMessageOr<void> result =
+      TryAddSymbolPath(std::filesystem::path{directory.toStdString()}, std::move(metric));
```

Comment:
I would prefer to NOT pass `ScopedMetric` into `TryAddSymbolPath` and set the metric status in `OnAddFolderButtonClicked` and other callers of `TryAddSymbolPath`.

Reply:
I fully agree, good call! I will change it

Reply:
done